src/app/util/Config/config.js

```diff
@@ -0,0 +1,249 @@
+export default [
```

Comment:
Can you check if there is a region ID map? Not sure if it exists.

Reply:
Actually, it seems Magento handles this some-how different. Please investigate. Potentially the regions could be disabled for countries, but this is not deafult values. We must do something with it...

Reply:
Country list is now taken from magento. If you want to change region to select, there is the possibility to get available_regions from country. That should be a separate ticket.